source/iface/common-byval.rst

```diff
@@ -4,24 +4,135 @@
 
 .. _common-byval:
 
-====================
- By-value Semantics
-====================
+*************************
+Common by-value Semantics
+*************************
 
-Types: :ref:`id`, :ref:`range`, :ref:`item`, :ref:`nd_item`,
-:ref:`h_item`, :ref:`group`, `sub_group`, and :ref:`nd_range`.
+.. Add sup_group ref after creating Data Access chapter structure
 
-Classes with value semantics support the following methods.
+Each of the following SYCL runtime classes:
+:ref:`id`, :ref:`range`, :ref:`item`, :ref:`nd_item`,
+:ref:`h_item`, :ref:`group`, `sub_group`, and :ref:`nd_range`
+must follow the following statements, where ``T`` is the
+runtime class type:
+
+* ``T`` must be default copy constructible and copy assignable
+  on the host application (in the case where ``T`` is available on
+  the host) and within SYCL kernel functions.
+* ``T`` must be default destructible on the host application
```

Comment:
```suggestion
* ``T`` must be default destructible in the host application
```

Reply:
Fixed in https://github.com/KhronosGroup/SYCL_Reference/pull/89/commits/473d00ab2ea82f9680cc852359be058ff674f8aa.